tests/test_auto_fit.py

```diff
@@ -250,7 +250,7 @@ def test_gaussian_2d_fitting(self):
         y = Variable('y')
         g = Variable('g')
 
-        model = Model({g: A * Gaussian(x, x0, sig_x) * Gaussian(y, y0, sig_y)})
+        model = GradientModel({g: A * Gaussian(x, x0, sig_x) * Gaussian(y, y0, sig_y)})
```

Comment:
Why? And shouldn't this be another PR?

Reply:
From my last comment:

> As a small aside, I have decided not to print the DiracDelta anymore for fear of false positives. It is much more useful that users get an error about this delta function being there, so they can reconsider what needs to be done in the first place since you probably shouldn't be having them. I therefore changed the relevant tests to use GradientModel instead of Model. I do think this might also be an issue in sympy, since mathematica does not give delta peaks for second derivatives of Gaussian functions.